src/components/templates/TemplateSearch.jsx

```diff
@@ -41,8 +41,8 @@ const TemplateSearch = () => {
         <div className="col">
           <form className="form-inline" onSubmit={event => event.preventDefault()}>
             <div className="form-group" style={{ paddingBottom: '10px', paddingTop: '10px' }}>
-              <label htmlFor="searchInput">Find a resource template</label>&nbsp;
-              <div className="input-group" style={{ width: '750px', paddingLeft: '5px' }}>
+              <label className="font-weight-bold" htmlFor="searchInput">Find a resource template</label>&nbsp;
+              <div className="input-group" style={{ width: '500px', paddingLeft: '5px' }}>
```

Comment:
Can you set this back to 750px?

Reply:
yes, though i think its better to not force it that wide (when you make the viewport smaller, it looks worse)